### DISCUSSION TITLE
Scrollback buffer disappears?

### DISCUSSION THREAD

**Discussion by jenstroeger** [kovidgoyal/kitty#7720] [General]
I’ve been enjoying the [scrollback buffer](https://sw.kovidgoyal.net/kitty/overview/#the-scrollback-buffer) without any issues for a while now — I can scroll up through the history using the mouse.

Today, however, that behavior suddenly changed after running a command. The scrollback history works, and after running `pytest` the block cursor disappears and scrolling using the mouse runs through previous commands in the command line rather than showing the scrollback buffer?

I can’t quite figure out what’s going on 😳 Any suggestions?

**Comment by kovidgoyal** [comment]
Some program running you ran grabbed the mouse and then didnt release
it on exit. Reset the terminal (ctrl+shift+delete) and you will be fine. And you will still
be able to scroll using the keyboard shortcuts for it.

**Reply by jenstroeger** [reply]
Thank you! I think what happened was that I ran `pytest` with a debugger for failing tests ([docs](https://docs.pytest.org/en/latest/how-to/failures.html#using-python-library-pdb-with-pytest)) and I used pudb ([link](https://github.com/inducer/pudb)) and that failed to start ([urwid](https://urwid.org/) issue) and didn’t release the UI resources it had already grabbed. Removing the debugging doesn’t cause the problem.

However, on my Mac the ctrl+shift+delete shortcut didn’t work?

**Comment by kovidgoyal** [comment]
See https://sw.kovidgoyal.net/kitty/conf/#shortcut-kitty.Reset-the-terminal

for mac specific shortcut.
